modules/ROOT/pages/index.adoc

```diff
@@ -1,39 +1,205 @@
+flow, a Mule app receives a message through an Event source (an endpoint, such
+as an HTTP listener), transforms that message into a new format, and processes
+any business logic before writing the processed message to an external system
+in a format that the system can read.
 
-Each Mule event processor can inform the runtime if it is CPU intensive, CPU light, or an IO intensive operation. This helps the runtime self-tune for different workloads dynamically, removing the need for you to manage thread pools manually to achieve optimum performance.
+//TODO: VERIFY INFO ON WAYS FLOW CAN CALL OTHER FLOWS
+To break up processing into more manageable units, Mule apps often contain
+multiple, interrelated flows, instead of just a single flow. A flow can call
+another flow as a direct reference, or it can call a flow through a common communication protocol or method, such as JMS, HTTP, FTP, or File. The called flows might be part of the same Mule app or might run in a separate deployment on another Mule runtime across the network.
 
-== About Thread Pool Executor (Schedulers) Configuration
+For more on this topic, see xref:about-flows[Flows and Subflows] and
+xref:mule-application-about.adoc[Mule Applications].
 
-You can configure schedulers (thread pool executors) through the well-commented `schedulers.conf` file in the standalone Mule EE installation.
+== Event Sources
 
-== See Also
+An Event source (or Trigger) is the first processor in a flow. This 
+processor receives a triggering event and creates a corresponding Mule event
+object to forward to the next event processing component in the flow.
 
-* xref:mule-application-about.adoc[Mule Applications]
+External clients can trigger processing in a Mule flow through a number of
+communication protocols and methods, such as JMS, HTTP, FTP, JDBC, or File.
+Mule translates these communication protocols and methods into a standard
+Message format, which passes through the flow's Event processors.
+
+Event sources in Mule can connect to specific external sources, either through
+a standard protocol or a third-party API. It is also possible to set a Scheduler.
+Some schedulers can poll for specific changes to external resources, such as new files or table rows in an external resource. Examples of listeners and connector operations that can trigger a flow include these:
+
+* HTTP, JMS, VM listeners in their associated connectors
+* On Table Row operation in the Database connector
+* On New or Updated File operation in the File and FTP connectors
+* Scheduler
+
+Note that you can also create your own Event source using the
+xref:1.1@mule-sdk::index.adoc[Mule SDK].
+
+== Event Processors
+
+Once a flow is triggered through an Event source, subsequent components process
+the Mule event as it travels through the flow. By default, each Event processor
+that receives a Mule event returns a new Mule message, typically with the set
+of attributes, message payload, and any variables that the processor returns.
```

Comment:
Processors do not return variables, only a payload and attributes.

Reply:
removed variables